test/test_shared.js

```
@@ -105,6 +176,97 @@ function loadClassFile(filePath) {
+  }
+
+  // Load and evaluate the Utils class
+  const utilsPath = path.join(
+    __dirname,
+    '../chrome_manifest_v3/class_utils.js'
+  );
+  const utilsCode = fs.readFileSync(utilsPath, 'utf8');
+
+  // Create local reference for eval scope
+  var G2T = global.G2T;
+  eval(utilsCode);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)